manifests/f/filips/FirefoxPWA/2.6.0/filips.FirefoxPWA.installer.yaml

```
@@ -0,0 +1,31 @@
+# Created with Komac v1.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: filips.FirefoxPWA
+PackageVersion: 2.6.0
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0

```

```suggestion
MinimumOSVersion: 10.0.0.0
Scope: machine
ElevationRequirement: elevatesSelf
```
Moving the common fields to root of the manifest for easier readability